wapitiCore/net/crawler.py

```diff
@@ -1154,6 +1202,58 @@ def auth_method(self, value):
             from requests_kerberos import HTTPKerberosAuth
             self._session.auth = HTTPKerberosAuth()
 
+    def try_login(self, auth_url: str):
+        """Try to authenticate with the provided url and credentials."""
+        if len(self._auth_credentials) != 2:
+            print(_("Login failed"))
+            return
+
+        username, password = self._auth_credentials
+        headers = {
+            'User-Agent': self.user_agent,
+            'Accept': 'text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,*/*;q=0.8',
```

Comment:
Those `Accept*` headers should be standard for the crawler session if not already present.

I saw some case where an XSS won't happen if html is not mentionned in `Accept`

Reply:
Added in Crawler __init__ method